src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/summaries/temperature/NodeLevelDimensionalSummary.java

```diff
@@ -347,9 +322,11 @@ public String getTableName() {
         public List<Object> getSqlValue() {
             List<Object> values = new ArrayList<>();
             values.add(myZone.name());
-            values.add(minShard == null ? "" : minShard.toJson());
-            values.add(maxShard == null ? "" : maxShard.toJson());
             JsonArray array = new JsonArray();
+
+            // The reason we are not getting shards ordered by temperature as
+            // we don't want to pay the cost of sorting while writes. We do writes
+            // more often than reads and therefore, we would rather sort on reads.
             for (ShardProfileSummary shard : shardProfileSummaries) {
```

Comment:
Are we making this schemaless by adding the entire json blob into a single tuple in SQL table ?